GVRf/Framework/src/org/gearvrf/GVRContext.java

```diff
@@ -1345,6 +1345,11 @@ protected synchronized static void resetOnRestart() {
      * same thread that calls this method, and it will not be either the GUI or
      * the GL thread.
      * 
+     * User should be responsible to wait until the last
```

Comment:
This is a good point, but it could perhaps be phrased a bit better.

> Users should not start a screenshot until any previous screen callbacks have returned. Overlapping screenshots will cause heavy load that could result in missed frames and/or out of memory errors.